microservices/OBJECT-IDENTIFIER-STRATEGY.md

```diff
@@ -0,0 +1,140 @@
+# Microservice Object Identifer Strategy
```

Comment:
Files shouldn't be SHOUTY named - the ones in the base are as they're the ones that normally are by convention (README, CONTRIBUTING et al) - everything else should be more sanely named :)

Reply:
Haha, fair point, I wasn't sure about that! I'll change it.